online-banking-cz/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) 2022 Adyen N.V.
+  ~
+  ~ This file is open source and available under the MIT license. See the LICENSE file for more info.
+  ~
+  ~ Created by atef on 25/8/2022.
+  -->
+
+<resources>
+
+
+    <style name="AdyenCheckout.Issuer" />
+
+    <style name="AdyenCheckout.Issuer.TermsAndConditions">
+        <item name="android:text">@string/checkout_issuers_list_terms_and_conditions_text</item>
+        <item name="android:layout_marginStart">@dimen/termsAndConditions_marginStart</item>
+        <item name="android:textColorLink">@color/termsAndConditionsColor</item>
```

Comment:
It would be better to define a generic `textColorLink` in the base theme. That would make it easier for merchants to override it